README.md

```diff
@@ -1,14 +1,12 @@
 # ❌ cargo-deny
 
-[![Build Status](https://github.com/EmbarkStudios/cargo-deny/workflows/CI/badge.svg)](https://github.com/EmbarkStudios/cargo-deny/actions?workflow=CI)
-[![Latest version](https://img.shields.io/crates/v/cargo-deny.svg)](https://crates.io/crates/cargo-deny)
+[![Embark](https://img.shields.io/badge/embark-open%20source-blueviolet.svg)](https://embark.dev)
+[![Embark](https://img.shields.io/badge/discord-ark-%237289da.svg?logo=discord)](https://discord.gg/dAuKfZS)
+[![Crates.io](https://img.shields.io/crates/v/cargo-deny.svg)](https://crates.io/crates/cargo-deny)
 [![Docs](https://img.shields.io/badge/The%20Book-📕-brightgreen.svg)](https://embarkstudios.github.io/cargo-deny/)
 [![API Docs](https://docs.rs/cargo-deny/badge.svg)](https://docs.rs/cargo-deny)
-[![MSRV](https://img.shields.io/badge/Rust-1.43-blue?color=fc8d62&logo=rust)](https://blog.rust-lang.org/2020/04/23/Rust-1.43.0.html)
-[![SPDX Version](https://img.shields.io/badge/SPDX%20Version-3.7-blue.svg)](https://spdx.org/licenses/)
```

Comment:
I'd like to keep these badges, MSRV is actually important for some users especially for a binary they install, and the SPDX version is a core part of how licenses are parsed and validated which is also important.

Reply:
OK. Is this something we want on the other repos too? Might be worth having a discussion in https://github.com/EmbarkStudios/opensource-template/issues/20 ?

Reply:
It doesn't come up very often, I think it can be something we fix if someone files an issue on it, but so far that's only happened on this repo.

Reply:
OK all done!